packages/apollo/README.md

```diff
@@ -46,11 +46,11 @@ USAGE
 * [`apollo plugins:link PLUGIN`](#apollo-pluginslink-plugin)
 * [`apollo plugins:uninstall PLUGIN...`](#apollo-pluginsuninstall-plugin)
 * [`apollo plugins:update`](#apollo-pluginsupdate)
-* [`apollo service:check`](#apollo-servicecheck)
-* [`apollo service:delete`](#apollo-servicedelete)
-* [`apollo service:download OUTPUT`](#apollo-servicedownload-output)
-* [`apollo service:list`](#apollo-servicelist)
-* [`apollo service:push`](#apollo-servicepush)
+* [`apollo service:check`](#apollo-servicecheck) [DEPRECATED]
+* [`apollo service:delete`](#apollo-servicedelete) [DEPRECATED]
+* [`apollo service:download OUTPUT`](#apollo-servicedownload-output) [DEPRECATED]
+* [`apollo service:list`](#apollo-servicelist) [DEPRECATED]
+* [`apollo service:push`](#apollo-servicepush) [DEPRECATED]
```

Comment:
I'm afraid this entire `README.md` is generated. 😬 

https://github.com/apollographql/apollo-tooling/blob/4f32ccc3083d9adfb7849a1427d696027b6259af/packages/apollo/package.json#L27

I _suppose_ that could go away? 🤭

Reply:
Haha - I had a feeling it was, but didn't see any references to it being auto-generated 🤦. I'll take a closer look. Thanks @abernix!

Reply:
@abernix I auto-generated the `README.md` and while the formatting isn't quite what I would have wanted, it's not that bad. It duplicates the deprecated messaging a bit extra (which isn't the end of the world), and it removes the deprecated label from the list of commands (since that list is generated from the command names). All in all though, it's not horrible, and the deprecated messaging is definitely visible. I've rebased and modified https://github.com/apollographql/apollo-tooling/pull/2308/commits/602c3a9ad1167d8e7045df456878d4cf13dd1c9c directly if you want to see the changes. Let me know if this is okay - thanks!

Reply:
Looks great to me.